packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -688,7 +688,12 @@ abstract class Element implements BuildContext {
   Widget get widget => _widget;
   Widget _widget;
 
-  bool _active = false;
+  RenderObjectOwner _owner;
+  RenderObjectOwner get owner => _owner;
```

Comment:
I was expecting to see this field used for the dirty elements list.  Maybe that's for a later patch?  (It's fine to delay that part of the change to a later patch.)